File: Fundamentals of C programming/step10/10.3/10.3.03.c
/*

Написать функцию void minmax(int * x, int * y), записывающую в переменную x минимальное из значений x и y, а в переменную y -- максимальное из этих значений.

Sample Input:

15 4

Sample Output:

4 15

*/

void minmax(int * x, int * y){
    int min;
    int max;
    if (*x<*y)
        min = *x;
        max = *y;
    if (*y<*x)
        min = *y;
        max = *x;
    
    *x = min;
    *y = max;
}

File: Good, good Python/step 10/10.03/10.3.03.py
'''

Подвиг 4. Вводится список названий городов в одну строчку через пробел. Выберите из этого списка один город случайным образом и отобразите его на экране.

Sample Input:

Тула Казань Смоленск Семипалатинск Уфа Москва Самара

Sample Output:

Казань

'''

import random
# установка "зерна" датчика случайных чисел, чтобы получались одни и те же случайные величины
random.seed(1)

# здесь продолжайте программу
s = input().split()
print(random.choice(s))
